genologics/entities.py

```diff
@@ -450,11 +456,11 @@ class Sample(Entity):
 
 
     @classmethod
-    def create(cls, lims, container, position, **kwargs):
+    def create(cls, lims, container, position, udfs, **kwargs):
```

Comment:
So is it ok to not have a pre defined value here for *udfs* ? I guess that is the reason why *travis* fails too

Reply:
Yes, I did add the default to None and switch to a dict